src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = ({ ...state });
+  const arrayFromState = [];
```

Comment:
The "arrayFromState" name doesn't match what actually this array is. statesHistory or statesLogs.